### PR TITLE
Replace db.Save with StrictUpdate

### PR DESCRIPTION
**Pull request by askurydzin**
* Introduces a breaking changes: signature of the DefaultApplyFieldMask changed to exclude ormObj.

 * DefaultApplyFieldMask should not remove associations
 * Instead, DefaultStrictUpdate should be called after FieldMask application.

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 [[constraint]]
   name = "github.com/grpc-ecosystem/grpc-gateway"
-  version = "1.3.1"
+  version = "1.4.1"
```
Comment (Calebjh) [inline Gopkg.toml]: How much of a difference does this make?

File: plugin/handlergen.go [hunk 2]
```diff
@@ -270,29 +259,8 @@ func (p *OrmPlugin) generatePatchHandler(message *generator.Descriptor) {
...
-	p.P(`return nil, err`)
-	p.P(`}`)
-
-	// convert ormObj to pbObj again (sic!) to trigger any logic that was
-	// written for AfterToPB/BeforeToPB.
-	p.P(`pbObj, err = ormObj.ToPB(ctx)`)
-	p.P(`if err != nil {`)
-	p.P(`return nil, err`)
-	p.P(`}`)
+	p.P(`return DefaultStrictUpdate`, typeName, `(ctx, &pbObj, db)`)
```
Comment (Calebjh) [inline plugin/handlergen.go]: Definitely cleaner (and also fixes the issue of removing associations in the ApplyFieldMask method). Have you tested it in practice to make sure it works correctly like this?

Comment (askurydzin) [reply plugin/handlergen.go]: build passes, and only thing that has changed is apply field mask is now not removes associations (the're all are removed in StrictUpdate)

File: Gopkg.toml (hunk 1 above)
Comment (askurydzin) [reply Gopkg.toml]: it varies from repo to repo, however I would make it consistent with gentool

**Comment by askurydzin** [comment]
noticed, that ormObj is no longer needed.

**Pull request merged**